porch/repository/pkg/git/draft.go

```diff
@@ -51,7 +53,21 @@ func (d *gitPackageDraft) UpdateResources(ctx context.Context, new *v1alpha1.Pac
 		ch.storeFile(path.Join(d.path, k), v)
 	}
 
-	message := fmt.Sprintf("Intermittent commit: %s", change.Type)
+	var message string
+	if change != nil {
+		message = fmt.Sprintf("Intermittent commit: %s", change.Type)
```

Comment:
Self-reviewing, and wondering if this should be "Intermediate commit..."

Reply:
Yes! sorry ...